node/building_blocks.re

```diff
@@ -67,7 +114,11 @@ let is_signable = (state, block) => {
   && !is_signed_by_self(state, ~hash=block.hash)
   && is_current_producer(state, ~key=block.author)
   && !has_next_block_to_apply(state, ~hash=block.hash)
-  && all_main_ops_are_known;
+  && all_main_ops_are_known
+  && contains_only_trusted_add_validator_op(
+       true,
```

Comment:
This doesn't make sense here.

Reply:
Can you please elaborate?

Reply:
Moved to `Node.apply_block`, before `Protocol.apply_block`